### ISSUE TITLE
Usage react-url-query with react-router v4 and react-router-redux

### ISSUE THREAD

**Issue by kyzia551** [pbeshai/react-url-query#17]
Hi! 
I'm trying to use react-url-query with react-router v4 and react-router-redux

My app component:
```javascript
import createHistory from 'history/createBrowserHistory';

const history = createHistory();
const initialState = window.REDUX_INITIAL_STATE || {};
const store = createStore(rootReducer, initialState, compose(
      applyMiddleware(thunk, routerMiddleware(history), urlQueryMiddleware()));

const component = (
  <Provider store={store}>
     <ConnectedRouter history={history}>
        <RouterToUrlQuery>
           {renderRoutes(routes)}
        </RouterToUrlQuery>
    </ConnectedRouter>
  </Provider>
);
```
When I'm dispatching this action 
```javascript
import { urlPushInAction, UrlQueryParamTypes } from 'react-url-query';

const changePageNumberInUrl = urlPushInAction(CHANGE_PAGE_NUMBER_IN_URL, 'page', 
                                              UrlQueryParamTypes.number);

function mapDispatchToProps(dispatch) {
  return {
    onChangePage: page => dispatch(changePageNumberInUrl(page))
  };
}
```
I see:
**Uncaught TypeError: _urlQueryConfig2.default.history.push is not a function**

**Comment by pbeshai** [comment]
It appears the v4 API has changed since I supported it. Investigating now.

**Comment by pbeshai** [comment]
Fixed in 1.1.2
